.github/workflows/ci.yml

```diff
@@ -70,12 +70,18 @@ jobs:
       run: |
         go build -v ./...
         go test -v ./...
+    - name: Build/Test launcher
```

Comment:
I also think this workflow needs to test its protobuf generation (to make sure things are up-to-date)

Reply:
added